lazyflow/operators/ioOperators/opInputDataReader.py

```diff
@@ -286,6 +288,18 @@ def _attemptOpenAsUfmf(self, filePath):
         else:
             return ([], None)
 
+    def _attemptOpenAsOmeZarrMultiscaleFileset(self, filePath):
```

Comment:
Does the `FileSet` bit in the name communicate some important information?

Reply:
Guess not? 😅 Some of the other methods are named -Volume and some -Fileset, I guess I preferred the latter. Will remove